### PR TITLE
Change tests to use import circt.stage.ChiselStage

### PR DESCRIPTION
**Pull request by chick**
Change tests to use import circt.stage.ChiselStage instead of
 chisel3.stage.ChiselStage. These are the tests that are trivially done via
emitChirrtl -> emitChirrtl
emitFirrtl  -> emitChirrtl
emitVerilog -> emitSystemVerilog

### Contributor Checklist

- [x] Did you add Scaladoc to every public function/method?
- [ ] Did you add at least one test demonstrating the PR?
- [x] Did you delete any extraneous printlns/debugging code?
- [x] Did you specify the type of improvement?
- [x] Did you add appropriate documentation in `docs/src`?
- [ ] Did you state the API impact?
- [x] Did you specify the code generation impact?
- [x] Did you request a desired merge strategy?
- [x] Did you add text to be included in the Release Notes for this change?

#### Type of Improvement

- code refactoring

#### API Impact

Only affects tests, using the circt.stage API instead of chisel.stage

#### Desired Merge Strategy

- Squash: The PR will be squashed and merged (choose this if you have no preference.

#### Release Notes

Chisel3 unit tests are being moved from chisel.stage API to the circt.stage API. This is the first of two PR that will implement
this

### Reviewer Checklist (only modified by reviewer)
- [ ] Did you add the appropriate labels?
- [ ] Did you mark the proper milestone (Bug fix: `3.4.x`, [small] API extension: `3.5.x`, API modification or big change: `3.6.0`)?
- [ ] Did you review?
- [ ] Did you check whether all relevant Contributor checkboxes have been checked?
- [ ] Did you do one of the following when ready to merge:
  - [ ] Squash: You/ the contributor `Enable auto-merge (squash)`, clean up the commit message, and label with `Please Merge`.
  - [ ] Merge: Ensure that contributor has cleaned up their commit history, then merge with `Create a merge commit`.

### CODE REVIEW

File: src/main/scala/circt/stage/ChiselStage.scala [hunk 1]
```diff
@@ -50,6 +47,14 @@ object ChiselStage {
   /** Elaborate a Chisel circuit into a CHIRRTL string */
   def emitCHIRRTL(gen: => RawModule): String = chisel3.stage.ChiselStage.emitChirrtl(gen)
 
+  /** Return a CHIRRTL circuit for a Chisel module
+    *
+    * @param gen a call-by-name Chisel module
+    */
+  def convert(gen: => RawModule): firrtl.ir.Circuit = {
+    chisel3.stage.ChiselStage.convert(gen)
+  }
```
Comment (seldridge) [inline src/main/scala/circt/stage/ChiselStage.scala:56]: Nit: It may be better to copy the body of the method than to have `circt.stage.ChiselStage` call `chisel3.stage.ChiselStage`.

File: src/main/scala/circt/stage/ChiselStage.scala [hunk 2]
```diff
@@ -130,6 +137,16 @@ object ChiselStage {
...
+
+  /** Return a Chisel circuit for a Chisel module
+    *
+    * @param gen a call-by-name Chisel module
+    */
+  def elaborate(
+    gen: => RawModule
+  ): chisel3.internal.firrtl.Circuit = {
+    chisel3.stage.ChiselStage.elaborate(gen)
+  }
```
Comment (seldridge) [inline src/main/scala/circt/stage/ChiselStage.scala:149]: Nit: consider copying the method body over.

File: src/test/scala/chiselTests/BulkConnectSpec.scala [hunk 3]
```diff
@@ -2,14 +2,14 @@ package chiselTests
...
-import chisel3.stage.ChiselStage
+import circt.stage.ChiselStage
 import chisel3.testers.BasicTester
 
 import scala.annotation.nowarn
 
 class BulkConnectSpec extends ChiselPropSpec {
   property("Chisel connects should emit FIRRTL bulk connects when possible") {
-    val chirrtl = ChiselStage.emitChirrtl(new Module {
+    val chirrtl = ChiselStage.emitCHIRRTL(new Module {
```
Comment (seldridge) [inline src/test/scala/chiselTests/BulkConnectSpec.scala:12]: My choice of naming on the `emitCHIRRTL` vs. `emitChirrtl` may have been misplaced given Chisel naming conventions. Out-of-scope of this PR, but we should consider renaming (automatically) `emitCHIRRTL` to `emitChirrtl`.

File: src/test/scala/chiselTests/MixedVecSpec.scala [hunk 4]
```diff
@@ -293,7 +293,7 @@ class MixedVecSpec extends ChiselPropSpec with Utils {
       io.outMono := (io.inMono: Data)
       io.outBi <> io.inBi
     })
-    chirrtl should include("io.outMono <= io.inMono @[src/test/scala/chiselTests/MixedVecSpec.scala")
-    chirrtl should include("io.outBi <= io.inBi @[src/test/scala/chiselTests/MixedVecSpec.scala")
+    chirrtl should include("io.outMono <= io.inMono @")
+    chirrtl should include("io.outBi <= io.inBi @")
```
Comment (seldridge) [inline src/test/scala/chiselTests/MixedVecSpec.scala:297]: Consider further shortening to `io.inMono"` (ignoring a check of the source locator as that doesn't seem to be relevant to the test).

File: src/test/scala/chiselTests/WireSpec.scala [hunk 5]
```diff
@@ -29,7 +29,8 @@ class WireSpec extends ChiselFlatSpec {
       out := in & wire & wire2
     }
 
-    val chirrtl = ChiselStage.emitChirrtl(new Dummy)
+    val chirrtl = ChiselStage.emitCHIRRTL(new Dummy)
+    // This test can fail when run from IntelliJ because the source locator path can be different
     chirrtl should include("wire wire : UInt<1> @[src/test/scala/chiselTests/WireSpec.scala")
     chirrtl should include("wire wire2 : UInt<1> @[src/test/scala/chiselTests/WireSpec.scala")
```
Comment (seldridge) [inline src/test/scala/chiselTests/WireSpec.scala:35]: Can the source locator be removed instead?

File: src/test/scala/chiselTests/util/CatSpec.scala [hunk 6]
```diff
@@ -68,7 +67,8 @@ class CatSpec extends ChiselFlatSpec {
       // noPrefix to avoid `out` as prefix
       out := Cat(in)
     }
-    val chirrtl = ChiselStage.emitChirrtl(new MyModule)
+    val chirrtl = ChiselStage.emitCHIRRTL(new MyModule)
+    // This test can fail when run from IntelliJ because the source locator path can be different
     chirrtl should include("cat(in[0], in[1]) @[src/test/scala/chiselTests/util/CatSpec.scala")
```
Comment (seldridge) [inline src/test/scala/chiselTests/util/CatSpec.scala:72]: Nit: stripping the source locator seems better.

File: src/test/scala/chiselTests/util/CatSpec.scala [hunk 7]
```diff
@@ -79,7 +79,8 @@ class CatSpec extends ChiselFlatSpec {
       val out = IO(Output(UInt()))
       out := Cat(in(0), in(1))
     }
-    val chirrtl = ChiselStage.emitChirrtl(new MyModule)
+    val chirrtl = ChiselStage.emitCHIRRTL(new MyModule)
+    // This test can fail when run from IntelliJ because the source locator path can be different
```
Comment (seldridge) [inline src/test/scala/chiselTests/util/CatSpec.scala:83]: Nit: strip source locator instead.

**Review by seldridge — APPROVED** [review]
LGTM

Some nits. 

More major nit: it seems better to strip source locators which don't work in IntelliJ than to add a comment that they may fail.

**Review by azidar — APPROVED** [review]

File: src/test/scala/chiselTests/WireSpec.scala (hunk 5 above)
Comment (chick) [reply src/test/scala/chiselTests/WireSpec.scala:35]: will do in next commit

File: src/test/scala/chiselTests/util/CatSpec.scala (hunk 6 above)
Comment (chick) [reply src/test/scala/chiselTests/util/CatSpec.scala:72]: Will do in next commit

File: src/test/scala/chiselTests/util/CatSpec.scala (hunk 7 above)
Comment (chick) [reply src/test/scala/chiselTests/util/CatSpec.scala:83]: Will do in next commit
